### ISSUE TITLE
hwc buildpack release 2.1.1 is broken

### ISSUE THREAD

**Issue by sunjayBhatia** [cloudfoundry-incubator/hwc-buildpack#1]
Pushing an app with the hwc-buildpack version 2.1.1 yields the following:

```
Starting app SMOKES-1-APP-f648150b-a496-4e45-5699-33680228f16a in org CF-SMOKE-ORG / space CF-SMOKE-SPACE as admin...
Creating container
Downloading app package...
Successfully created container
Downloaded app package (353.8K)
Staging...
       **ERROR** Could not set file mode for C:\windows\TEMP\hwc034471491\hwc.zip
Failed to compile droplet
chmod C:\windows\TEMP\hwc034471491\hwc.zip: not supported by windows
       Copy [C:\containerizer\ACF6921809FD1F9517\user\tmp\buildpacks\9FD662F3ABC64D1B4E24E4769D82CAE7\dependencies\https___github.com_cloudfoundry-incubator_hwc_releases_download_1.0.1_hwc.zip]
Exit status 223
Staging failed: Exited with status 223
Destroying container
Successfully destroyed container

FAILED
BuildpackCompileFailed
```

Probably caused by these commits:
- https://github.com/cloudfoundry-incubator/hwc-buildpack/commit/a7dd9bd576fd68301f5004d5da20e1fdc658f92b
- https://github.com/cloudfoundry/libbuildpack/compare/cf7b4d1059e0cc712e1fda1f5755c6b61f0dccb3...bbff67a2e11360b9d5d8110cf7e11195b3946a9e#diff-3f1ad41879763caf285707f64678af17

Chmod [doesn't work on windows ](http://stackoverflow.com/questions/33445727/how-to-control-file-access-in-windows)

The stack in the manifest is correct:
```
cf_stacks:
  - windows2012R2
```

**Comment by sesmith177** [comment]
@sunjayBhatia This [commit on libbuildpack](https://github.com/cloudfoundry/libbuildpack/commit/10ea613134d81daf00581b87df2eb57812169568) removes the call to os.Chmod(). We updated libbuildpack in the HWC buildpack here: 8bc9688c367dec2dd45831f299fc1796df88dbb7

**Comment by sclevine** [comment]
Closing this because it appears to be fixed by a revert in v2.1.2, and the latest libbuildpack will be available in v2.1.3.
